app/cdap/components/DrawerWidget/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Dispatch, ReactNode, SetStateAction } from 'react';
+
+export interface IHeaderText {
+  headingText: string | number | ReactNode | boolean;
+}
+
+export interface IDrawerWidgetProps {
+  headingText?: string | ReactNode;
+  openDrawer: boolean | Dispatch<SetStateAction<boolean>> | unknown;
+  showDivider?: boolean;
+  headerActionTemplate?: unknown;
+  children?: unknown[] | boolean;
+  closeClickHandler?: unknown;
+  showBackIcon?: boolean;
+  anchor?: unknown;
```

Comment:
try to replace unknown types with proper type whereever possible

Reply:
done